spec/components/big_number_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+describe "Big number", type: :view do
+  def component_name
+    "big_number"
+  end
+
+  it "renders nothing when nothing is passed to it" do
+    assert_empty render_component({})
+    assert_empty render_component(number: false)
+  end
+
+  it "renders a number value" do
+    render_component(number: 500)
+
+    assert_select ".gem-c-big-number", count: 1
+    assert_select ".govuk-list", count: 0
```

Comment:
What's this check for? I don't see the `govuk-list` class used anywhere in the component.

Reply:
Oops, legacy test. Removing...